src/common/values.ts

```diff
@@ -36,11 +36,11 @@ export const getK8sVersion = (): string => {
  * Find what image tag is defined in configuration for otomi
  * @returns string
  */
-export const getImageTag = async (): Promise<string> => {
+export const getImageTag = async (isBootstrap = false): Promise<string> => {
   if (process.env.OTOMI_TAG) return process.env.OTOMI_TAG
+  if (isBootstrap) return `v${pkg.version}`
```

Comment:
what if the values indicate a certain version?

Reply:
there are no values when bootstrap asks for it and this is only for reporting feedback, so trivial

Reply:
no user shall see discrepancy between release and dev image, only developers...